Extensions/people_notifier.js

```diff
@@ -462,6 +462,7 @@ XKit.extensions.people_notifier = new Object({
 						"Error message: <p>" + e.message + "</p>" +
 						"Please try again later or file a bug report." +
 						"error",
+						undefined,
```

Comment:
incorrect fix - the ` +` two lines above is a mistake, the `"error"` should be defining the type of window